mdrst.rst

```diff
@@ -2,15 +2,14 @@
 RST cheat sheet for Markdown fans
 #################################
 
-This is a comparison of Markdown and ReStructured Text to help people
-authoring or editing RST files.
+This cheatsheet provides a quick reference for translating
+Markdown to ReStructured Text (RST) to help someone author or edit
+RST files. 
 
 You can look at the `original Markdown file <md.md>`_ and the
```

Comment:
This could possibly be moved to `Additional Resources` at the bottom to more quickly get into the cheatsheet.